scanpipe/models.py

```diff
@@ -2383,6 +2384,18 @@ def macho_binaries(self):
     def executable_binaries(self):
         return self.union(self.win_exes(), self.macho_binaries(), self.elfs())
 
+    def with_children(self, project):
```

Comment:
The name is confusing because it does not include the children records, it includes "has_children".
A better name would be `with_has_children`.